File: rust-notebook/src/2026/01/06/src/main.rs
fn main() {
    println!("Hello, world!");
    println!("{}", url());
}

pub fn url() -> String {
    String::from("https://ontouchstart.github.io/rust-notebook/book/2026/01/06")
}

#[test]
fn test_url() {
    assert_eq!(
        url(),
        String::from("https://ontouchstart.github.io/rust-notebook/book/2026/01/06")
    )
}

File: rust-notebook/src/2026/01/06/main.py
import ontouchstart

name = "notebook-2026-01-06"


def url():
    return "https://ontouchstart.github.io/rust-notebook/book/2026/01/06"


def test_url():
    assert url() == "https://ontouchstart.github.io/rust-notebook/book/2026/01/06"


def test_ontouchstart_url():
    assert (
        ontouchstart.url()
        == "https://github.com/ontouchstart/ontouchstart.github.io/tree/2026_01_06_py"
    )


def main():
    print("Hello, world!")
    print(url())
    print(ontouchstart.url())


if __name__ == "__main__":
    main()
